pennylane/spin/lattice_shapes.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains functions to create 
+:class:`~pennylane.spin.lattice` objects of different shapes.
+"""
+
+from pennylane import numpy as np
+
+from .lattice import Lattice
+
+
+def Chain(n_cells, boundary_condition=False, neighbour_order=1):
```

Comment:
Might want to at least have a developer note on top to capture the input and outputs for the `signatures` of all the classes together.